src/content/blog/drupalcon-barcelona-2024.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "DrupalCon Barcelona Wrap-up"
+pubDate: 2024-09-30
+#modifiedDate: 2024-09-06
+summary: DDEV happenings at DrupalCon Barcelona
+author: Randy Fay
+#featureImage:
+#  src: /img/blog/2024/09/ddev-blog.png
+#  alt: DDEV running in the GitLab Pipeline
+#  credit: "[Milad Fakurian](https://unsplash.com/de/@fakurian) (Background)"
+categories:
+  - Community
+---
+
+I spent the last week in Barcelona, Spain for DrupalCon Barcelona. It was so great to meet old friends and new, and to workk with all to integrate new contributors to Drupal.
+
+## BoFs (Birds-of-a-feature) sessions
+
+A cool thing about DrupalCon is that anybody can set up a mini-session, and no real approval process is needed. I typically propose several of these, since they're a great way to deal with niche topics with a few people and have nice, smaller conversations with people who have specific needs. This time I did:
+
+* **Working together: DDEV on Gitpod and GitHub Codespaces**: We talked about the many ways to use DDEV in Cloud IDEs like Gitpod and GitHub Codespaces. People were really interested in this, since of course these approaches don't require a local development environment at all (or rather, the local environment is in the cloud on a VM, and presented via a browser). See the DDEV docs on [Gitpod](https://ddev.readthedocs.io/en/latest/users/install/ddev-installation/#ddev-installation-gitpod) and [GitHub Codespaces](https://ddev.readthedocs.io/en/latest/users/install/ddev-installation/#ddev-installation-codespaces). 
+* **Git on the Command Line - Office Hours**: We had fun talking about the basics of Git, what people need to know to really master it and do their work predictably and without fear. This one was the first thing in the morning, so had a smaller turnout.
+* **DDEV Office Hours**: At most DrupalCons we do this simple free-form BoF so that people can bring the issues and discussions they want to have. At this one we had some really strong and experienced DDEV users, so it ended up being that they were helping me to understand their workflow rather than me coaching them.
+* **Prepare for Contribution Day: Issue Forks, Drupalpod, DDEV, Git**: The highlight of DrupalCon is the Contribution Day, but sometimes coming to it it's a little intimidating because of Drupal's unusual contribution workflow. We talked about Drupal's issue workflow, how issue forks work, merge requests, etc. In the Drupal workflow, each issue has its own Git project fork, instead of the traditional open-source workflow where each person has a personal fork. In the Drupal workflow, anybody can push to any of the branches, or create a new branch. It's quite amazing.
```

Comment:
```suggestion
* **Prepare for Contribution Day: Issue Forks, DrupalPod, DDEV, Git**: The highlight of DrupalCon is the Contribution Day, but sometimes coming to it it's a little intimidating because of Drupal's unusual contribution workflow. We talked about Drupal's issue workflow, how issue forks work, merge requests, etc. In the Drupal workflow, each issue has its own Git project fork, instead of the traditional open-source workflow where each person has a personal fork. In the Drupal workflow, anybody can push to any of the branches, or create a new branch. It's quite amazing.
```